assets/apps/components/src/Controls/Textarea.js

```diff
@@ -0,0 +1,30 @@
+import { TextareaControl } from '@wordpress/components';
+import PropTypes from 'prop-types';
+
+const Textarea = ({ label, value, onChange, rows }) => {
```

Comment:
How about we try to simplify this and keep the functionality within our TextareaComponent? We might be adding too much complexity to what should be a straightforward text area. By utilizing the TextareaControl from WP within our TextareaComponent, we can keep things clean and organized.